src/routes/InstancesRoute.js

```diff
@@ -72,20 +79,26 @@ class InstancesRoute extends React.Component {
       createInstance,
     } = this.props;
 
+
     if (isLoading()) {
       return null;
     }
 
+    const data = getData();
+
     return (
       <InstancesView
         parentResources={resources}
         parentMutator={mutator}
-        data={getData()}
+        data={data}
```

Comment:
What's the benefit in separating `data` into a variable?

Reply:
This was a left over from my debugging data :) Will clean it up. Thanks for catching it!